README.md

```diff
@@ -64,7 +69,11 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+var doSomething = function(thingToDo){
+  console.log("You need to " + thingtToDo );
+}
+
+doSomething(vacuum);
 ```
```

Comment:
not exactly. What we want to do here is use `thingToDo` argument as a function. It should be referenced where you declare it as an argument, and it should be invoked in the actual function call.